src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -248,6 +248,28 @@ public void disabledPeriodic() {
 			isArmCoast = !isArmCoast;
 		}
 		wasArmButtonPressed = isArmButtonPressed;
+
+		if ((subsystems.visionSubsystem != null) && (subsystems.armLedSubsystem != null)) {
+			// spotless:off
+			// Check if there is a recent target, and if the angle is
+			boolean isAlignmentCorrect =
+					subsystems.visionSubsystem.hasTargets()
+					&& ((subsystems.visionSubsystem.getLastTimestampSeconds() - Timer.getFPGATimestamp())
```

Comment:
Looking at how the pose estimator is used in the photon example [here](https://github.com/PhotonVision/photonvision/blob/master/photonlib-java-examples/apriltagExample/src/main/java/frc/robot/Drivetrain.java#L190) I think it is, although it would be worth paying attention to when testing.